src/components/SelectInputBlock/selectInputBlock.jsx

```diff
@@ -1,53 +0,0 @@
-/* eslint-disable react/destructuring-assignment */
-/* eslint-disable react/forbid-prop-types */
-import React from 'react';
-import PropTypes from 'prop-types';
-import Select from '@material-ui/core/Select';
-import { Typography } from '@material-ui/core';
-
-function SelectBlock(props) {
-  const { caption, itemArray, id, handleStateChange } = props;
-  // const selectId = `select-${id}`;
-
-  // const indexFromStorage = localStorage.getItem(selectId);
-  // const valueFromStorage = indexFromStorage === null ? itemArray[activeIndex] : +indexFromStorage;
-
-  // eslint-disable-next-line no-unused-vars
-  // const [state, setState] = React.useState({
-  //   item: valueFromStorage,
-  // });
-
-  // const handleChange = item => event => {
-  //   setState({
-  //     [item]: event.target.value,
-  //   });
-  //   localStorage.setItem(selectId, event.target.value);
-  // };
-  const optionsArray = [];
-  itemArray.forEach(element => {
-    const index = itemArray.indexOf(element);
```

Comment:
Callback function for `forEach` provides index as second parameter:
```
arr.forEach(function callback(currentValue, index, array) {
    //your iterator
}[, thisArg]); 
```
You do not need to get index from indexOf.